docs/_includes/conditional-include-directive.adoc

```diff
@@ -0,0 +1,85 @@
+////
+== Conditional Include Directive
+
+- User manual
+////
+You can include or exclude lines of text in your document from being processing by specifying three conditional include directives:
+
+* ifdef
+* ifndef
+* ifeval
```

Comment:
Under this list, add a paragraph briefly explaining what constitutes a condition.

> These conditions make decisions about whether to processes content based on facts.
The facts are typically the existence and/or value of document-level attributes (built-in/implicit), defined in the document or via the API or cli).
For example, you may only want to include (or not include) certain content when using the `html5` backend.
In order to do that, you'd check for the existence of the `backend-html5` attribute using a conditional include directive.
We'll cover specific examples in each section.